lib/vsc/install/commontest.py

```diff
@@ -122,7 +122,7 @@ def _import(self, pkg):
         except ImportError as e:
             log.warn('Import of %s failed with %s' % (pkg, e))
 
-        self.assertTrue(pkg in sys.modules)
+        self.assertTrue(pkg in sys.modules, msg='import %s was success' % pkg)
```

Comment:
how about doing an `assertTrue(False, ...)` in the `except` above, where you can include the error in the message?